modules/bitgo/src/v2/coinFactory.ts

```diff
@@ -57,6 +57,7 @@ import {
   Stx,
   Sui,
   Susd,
+  Ton,
```

Comment:
nit: please move it below to maintain the alphabetic order

Reply:
i put it here because the one below are all testnet token

Reply:
Actually I saw Trx after the testnet coins, that's why I thought so, but I am fine with this